README.md

```diff
@@ -559,6 +527,287 @@ You can build a pipeline to use [FHIR converter](https://github.com/microsoft/FH
+
+Once you have built the command line tool, you will find executable file Microsoft.Health.Dicom.Anonymizer.CommandLineTool.exe in the $SOURCE\DICOM\src\Microsoft.Health.Dicom.Anonymizer.CommandLineTool\bin\Debug|Release\netcoreapp3.1 folder.
+
+You can use this executable file to anonymize DICOM file.
+
+```
+> .\Microsoft.Health.Dicom.Anonymizer.CommandLineTool.exe -i myInputFile -o myOutputFile
+```
+
+### Use Command Line Tool
+The command-line tool can be used to anonymize one DICOM file or a folder containing DICOM files. Here are the parameters that the tool accepts:
+
+
+| Option | Name | Optionality | Default | Description |
+| ----- | ----- | ----- |----- |----- |
+| -i | inputFile | Required (for file conversion) | | Input DICOM file. |
+| -o | outputFile | Required (for file conversion) | |  Output DICOM file. |
+| -c | configFile | Optional |configuration.json | Anonymizer configuration file path. It reads the default file from the current directory. |
+| -I | inputFolder | Required (for folder conversion) |  | Input folder. |
+| -O | outputFolder | Required (for folder conversion) |  | Output folder. |
+| --validateInput | validateInput | Optional | false | Validate input DICOM file against value multiplicity, value types and format in DICOM specification. |
+| --validateOutput | validateOutput | Optional | false | Validate output DICOM file against value multiplicity, value types and format in DICOM specification. |
+
+> **[NOTE]**
+> To anonymize one DICOM file, inputFile and outputFile are required. To anonymize a DICOM folder, inputFolder and outputFolder are required.
+
+Example usage to anonymize DICOM files in a folder:
+```
+>.\Microsoft.Health.Dicom.Anonymizer.CommandLineTool.exe convert -I myInputFolder -O myOutputFolder -c myConfigFile
+```
+
+## Sample configuration file
+The configuration is specified in JSON format and has three required high-level sections. The first section named _rules_, it specifies anonymization methods for DICOM tag. The second and third sections are _defaultSettings_ and _customizedSettings_ which specify default settings and customized settings for anonymization methods respectively.
+
+|Fields|Description|
+|----|----|
+|rules|De-ID rules for tags.|
+|defaultSettings|Default settings for anonymization functions. Default settings will be used if not specify settings in rules.|
+|customSettings|Custom settings for anonymization functions.|
+
+
+DICOM Anonymization tool comes with a sample configuration file to help meet the requirements of HIPAA Safe Harbor Method. DICOM standard also describes attributes within a DICOM dataset that may potentially result in leakage of individually identifiable information according to HIPAA Safe Harbor. Our tool will build in a sample [configuration file]() that covers [application level confidentiality profile attributes](http://dicom.nema.org/medical/dicom/2018e/output/chtml/part15/chapter_E.html) defined in DICOM standard.
```

Comment:
add link here for configuration file.
DICOM/src/Microsoft.Health.Dicom.Anonymizer.CommandLineTool/configuration.json